packages/ibm-products/src/components/SimpleHeader/SimpleHeader.js

```diff
@@ -49,23 +52,25 @@ const BasicHeader = ({
   return (
     <header className={cx(blockClass, className)} {...rest}>
       {breadcrumbs?.length > 0 && (
-        <Breadcrumb
+        <BreadcrumbWithOverflow
           noTrailingSlash={noTrailingSlash}
           className={cx(`${blockClass}__breadcrumbs`)}
-        >
-          {breadcrumbs.map(({ key, label, href }) => (
-            <BreadcrumbItem key={key} href={href}>
-              {label}
-            </BreadcrumbItem>
-          ))}
-        </Breadcrumb>
+          breadcrumbs={breadcrumbs}
+          maxVisible={maxVisible}
+          overflowAriaLabel={overflowAriaLabel}
+        />
       )}
       {title && <h1 className={cx(`${blockClass}__title`)}>{title}</h1>}
     </header>
   );
 };
 
-BasicHeader.propTypes = {
+export const overview_arialabel_required_if_breadcrumbs_exist = isRequiredIf(
```

Comment:
Pretty minor change, but overview should be `overflow`?

```suggestion
export const overflowAriaLabel_required_if_breadcrumbs_exist = isRequiredIf(
```